rules/code-quality/tests/Rector/Isset_/IssetOnPropertyObjectToPropertyExistsRector/Fixture/skip_multiple_properties.php.inc

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Rector\CodeQuality\Tests\Rector\Isset_\IssetOnPropertyObjectToPropertyExistsRector\Fixture;
+
+class SomeClass3
+{
+    private $x;
+    private $y;
+
+    public function run()
+    {
+        isset($this->x, $this->y);
+    }
+}
```

Comment:
Just curious, why is this case skipped?

Reply:
If not skipped, previously, it will only change first parameter, so only changed to: 

```diff
+ property_exists($this, 'x') && $this->x !== null;
```

It may need to use `NodeVisitorAbstract` like in https://github.com/rectorphp/rector/blob/master/packages/node-type-resolver/src/NodeVisitor/StatementNodeVisitor.php but I've no idea how that works with `Isset_` yet. So the safest way currently is just skip it.

```php
            if ($previous && $previous->getAttribute(AttributeKey::PARENT_NODE) === $current) {
                continue;
            }

            if ($next && $next->getAttribute(AttributeKey::PARENT_NODE) === $current) {
                continue;
            }
```

@TomasVotruba do you have example for how to append multiple `BooleanAnd` expression?

Reply:
I see. We can handle this in 2nd iteration of this rule.

I'll merge this as it is now.